src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,174 @@
+package Duke;
```

Comment:
Overall I find everything looks really good, with JavaDocs well written and good testing, nice!